DQMOffline/Trigger/python/HiggsMonitoring_cff.py

```diff
@@ -269,6 +270,56 @@
 mu8diEle12CaloIdLTrackIdL_dz.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_Mu8_DiEle12_CaloIdL_TrackIdL_DZ_v*")
 mu8diEle12CaloIdLTrackIdL_dz.denGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_Mu8_DiEle12_CaloIdL_TrackIdL_v*")
 
+###########AK8 jet trigger with AK8 CSV requirement############
+AK8PFJet330_TrimMass30_CSV04_PromptMonitoring = hltBTVmonitoring.clone()
```

Comment:
why don't you clone the above module ?
(instead of having the same set of lines twice ?!?!?)

Reply:
Ahh that's a lot cleaner. I'll change that